config/locales/en.yml

```diff
@@ -43,7 +43,7 @@ en:
     original_consultation: Original consultation
     ran_from: This consultation ran from
     respond_online: Respond online
-    visit_soon: Visit this page again soon to download the outcome to this public&nbsp;feedback.
+    visit_soon: Visit this page again soon to download the outcome to this public feedback.
```

Comment:
The `&nbsp;` character removed here is present in all the other locales (as far as I can see). Most of them will probably never be viewed but I would guess at least the `cy` one would be. So strictly speaking I feel we should change some/all of these as well.